drivers/volume/portworx/portworx.go

```diff
@@ -1075,6 +1115,18 @@ func (d *portworx) getClusterManagerByAddress(addr string) (cluster.Cluster, err
 	return clusterclient.ClusterManager(cClient), nil
 }
 
+func (d *portworx) getClusterToken(n node.Node, op string) (*client.Response, error) {
+	url := d.constructURL(n.Addresses[0])
```

Comment:
There is an openstorage API for this

Reply:
@disrani-px openstorage build in current torpedo points to **v3.0.0-beta**, which is way behind current master branch. Since cluster pair api is added recently to openstorage we need to vendor in latest commit . Please let me know if it's ok, I have local commit already created for this changes